src/test/java/com/clickhouse/kafka/connect/sink/ClickHouseSinkTaskWithSchemaTest.java

```diff
@@ -439,18 +441,18 @@ public void schemaWithFixedStringMismatchTest() {
         ClickHouseHelperClient chc = createClient(props);
 
         String topic = "fixed-string-mismatch-table-test";
-        int fixedStringSize = RandomUtils.nextInt(1, 100);
+        int fixedStringSize = RandomUtils.nextInt(2, 100);
```

Comment:
Can we log it so if we encounter any issue here, we can reproduce easily

Reply:
In the test itself, if for some reason it fails and we want to reproduce it, we need to have the exact `fixedStringSize`